webapp/src/main/webapp/skin/less/webconferencing-default.less

```diff
@@ -1,4 +1,4 @@
-@import "/webconferencing/skin/webconferencing.css";
+@import "webconferencing.less";
```

Comment:
Does it make a sense to have this file if it just includes the other one ?

Reply:
Done!